auparse/zaudit_msg_types.go

```diff
@@ -147,6 +149,8 @@ const (
 	AUDIT_MAC_IPSEC_EVENT          AuditMessageType = 1415
 	AUDIT_MAC_UNLBL_STCADD         AuditMessageType = 1416
 	AUDIT_MAC_UNLBL_STCDEL         AuditMessageType = 1417
+	AUDIT_MAC_CALIPSO_ADD          AuditMessageType = 1418
+	AUDIT_MAC_CALIPSO_DEL          AuditMessageType = 1419
```

Comment:
I think you put this changes on the wrong commit by mistake

Reply:
I noticed these changes too. I ran `go generate` to update headers, but these two types got added. And I had not updated any of the URLs where the headers come from.

I'd say merge it as is because it's not going to hurt anything.

Reply:
Ok, merged